extensions/common/vault/vault-hashicorp/src/test/java/org/eclipse/edc/vault/hashicorp/client/HashicorpVaultClientIntegrationTest.java

```diff
@@ -37,22 +39,31 @@
 
 @ComponentTest
 @Testcontainers
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class HashicorpVaultClientIntegrationTest {
     @Container
-    static final VaultContainer<?> VAULT_CONTAINER = new VaultContainer<>("vault:1.9.6")
+    final VaultContainer<?> VAULT_CONTAINER = new VaultContainer<>(getVaultImage())
```

Comment:
please do not change that. it will spin up a separate container _for each test method_, which needlessly blows up test runtimes.